spec.md

```diff
@@ -445,9 +445,99 @@ Resource properties:
 * `name` - optional
 * Links to any metadata objects controlled by the DataPub server
 * Links to any metadata objects modifiable by the resource's controller
-### Resource Operations
 
-#### Resource HTTP API
+### Create Resource Operation
+
+A resource is created by making a Create Resource request to a given [=space=],
+and (optionally) a [=collection=]. If a collection is not explicitly specified,
```

Comment:
I think we can define this as only being sent to a collection iff

spaces are not collections.

iff the resource path ends in '/', it is a collection.

`/{space.id}` is the path to a space

`/{space.id}/` is the path to the root/default collection of that space

Reply:
POST /{space} is undefined but POST /{space}/ is defined as creating a resource in the space's default collection

Reply:
I like it. Will emphasize.